examples/example-tls/src/main/java/io/grpc/examples/helloworldtls/HelloWorldServerTls.java

```diff
@@ -110,11 +89,16 @@ public static void main(String[] args) throws IOException, InterruptedException
             System.exit(0);
         }
 
+        // If only providing a private key, you can use TlsServerCredentials.create() instead of
+        // interacting with the Builder.
+        TlsServerCredentials.Builder tlsBuilder = TlsServerCredentials.newBuilder()
+            .keyManager(new File(args[1]), new File(args[2]));
```

Comment:
Similar to my comment on the client side you can illustrate the `TlsServerCredentials.create()` use for the `args.length == 3` case explicitly. But just a suggestion.